rclcpp/include/rclcpp/node_options.hpp

```diff
@@ -258,6 +258,21 @@ class NodeOptions
   NodeOptions &
   clock_qos(const rclcpp::QoS & clock_qos);
 
+
+  /// Return the use_clock_thread flag.
+  RCLCPP_PUBLIC
+  bool
+  use_clock_thread() const;
+
+  /// Set the use_clock_thread flag, return this for parameter idiom.
+  /**
+   * If true, a dedicated thread will be used to subscribe to "/clock"
+   * topic, using a callback group
```

Comment:
Maybe?

```suggestion
   * If true, a dedicated thread will be used to subscribe to "/clock"
   * topic, using a callback group and executor dedicated to this task.
```

Seemed like an incomplete thought, and also no punctuation.

Reply:
Or just

```suggestion
   * If true, a dedicated thread will be used to subscribe to "/clock"
   * topic.
```

Since the details (beyond it being in a thread) maybe don't matter so much?

Reply:
Also, this should be mentioned in a higher doc block as well, around line 49 of this file.

Reply:
Done in [last commit](https://github.com/ros2/rclcpp/pull/1556/commits/e16e18430eceaa5647a8b9e3e0447ff367e2973e)